roles/vault_pki/README.md

```diff
@@ -47,4 +47,5 @@ Role variables
     * `vault_pki_write_certificates_host:` The host on which certificates will be written to. (default: "localhost")
     * `vault_pki_certificates_directory`: directory to output certificate files to.
     * `vault_pki_write_certificates`: whether to write generated certificates to a file or not (default: `false`)
```

Comment:
unrelated: I notice this variable isn't used

Reply:
Removed in a different PR